src/Screens/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './Homepage.css'
+
+export default function Homepage() {
+  return (
+    <div className="home-background">
+      <div>
+        <p className="home-title">The Homestead</p>
+        <h3 className="home-tagline">... that we might present every man complete in Christ!</h3>
+      </div>
+      <div>
+        <img src="http://www.homesteadretreats.com/About_Us_files/The%20Hall%20-%20original.jpg" alt="homestead-drawn" className="homestead"/>
+      </div>
+      <div className="homepage-link">
+        <button className="main-link"><Link to="">Enter Site</Link></button>
```

Comment:
This page was introduced in the original site since we wanted a very custom container for the flash site. Since resizing the window is much more complex, we opted to open a new window instead with the precise dimensions. Since this is no longer a flash site, this style of entry is no longer required; this page is not needed any longer.